src/agent/firebase-controller.ts

```diff
@@ -133,10 +133,11 @@ export class FirebaseController implements Controller {
 
     // Calculate the debuggee id as the hash of the object.
     // This MUST be consistent across all debuggee instances.
-    this.debuggeeId = crypto
+    const debuggeeHash = crypto
       .createHash('md5')
       .update(JSON.stringify(debuggee))
```

Comment:
Unrelated to this change, but something to note, it seems JSON.stringify may not be deterministic in the ordering of keys: https://github.com/auth0/node-jsonwebtoken/issues/404. It's possible in practice the way we end up using things and creating the debuggee it may not be an issue for us.

Reply:
I'll leave this as a TODO for now, but you're right.  The order appears to be based on insertion order which *should* be deterministic for these debuggee objects, but it would be good to be able to guarantee that it's stable.